### PR TITLE
Update queue.go

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
Remove `PILOT_THROTTLE` env

### CODE REVIEW

**Comment by costinm** [comment]
Why ?

File: install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml [hunk 1]
```diff
@@ -67,8 +67,6 @@ spec:
               fieldRef:
                 apiVersion: v1
                 fieldPath: metadata.namespace
-          - name: PILOT_THROTTLE
-            value: "500"
```
Comment (rshriram) [inline install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml]: @costinm ^^^ what was this env var for? I think we have way too may env vars in Pilot now, peppered all over the code. We need to consolidate them before we forget what is what. Or remove them.

**Comment by rshriram** [comment]
I think he removed it because you added a TODO to remove the variable after 0.7. If its trully unnecessary, we should get rid of it. I don't know how this behaves in concert with the rest of the knobs all over pilot.

**Comment by ZackButcher** [comment]
+1 on killing all the environment variables. Our logging framework supports setting verbosity per component, other stuff like this should be driven by flags. Env vars are evil.

**Comment by hzxuzhonghu** [comment]
rebased

**Comment by rshriram** [comment]
might be useful to send same Pr to releaes-1.0 branch

**Pull request merged**